code/script.js

```diff
@@ -32,20 +42,96 @@ const showMessage = (message, sender) => {
-  showMessage("Hello there, What's your name?", 'bot');
-  // Just to check it out, change 'bot' to 'user' here 👆
+const greeting = () => {
+  questionNumber = 1
+  botReply(`Howdy hungry human! What's your name?`)
 }
 
-// Set up your eventlisteners here
+const showFoodTypes = (msg) => {
+  questionNumber++
+  botReply(`Nice to meet you ${msg}! What kind of recipe are you looking for?`)
+
+  inputWrapper.innerHTML = `
+  <button id="pastaBtn">Pasta</button>
+  <button id="texmexBtn">TexMex</button>
+  <button id="pizzaBtn">Pizza</button>
+  <button id="vegBtn">Vegetarian</button>
+  ` 
+  document
+  .getElementById('pastaBtn')
+  .addEventListener('click', () => nextQuestion ('Pasta'))
+  document
+  .getElementById('texmexBtn')
+  .addEventListener('click', () => nextQuestion ('TexMex'))
+  document
+  .getElementById('pizzaBtn')
+  .addEventListener('click', () => nextQuestion ('Pizza'))
+  document
+  .getElementById('vegBtn')
+  .addEventListener('click', () => nextQuestion ('Vegetarian'))
+}
 
+const showRecipe = (food) => {
+  questionNumber++
+  
+  if (food === 'Pasta') {
+    botReply(`Pasta, huh? Molto bene! Please click on the dish you'd like to try.`)
+    inputWrapper.innerHTML = `
+    <button id="carBtn"><a href="https://www.simplyrecipes.com/recipes/spaghetti_alla_carbonara/" target="_blank">Pasta Carbonara</a></button>
+    <button id="fetBtn"><a href="https://www.simplyrecipes.com/recipes/fettuccine_with_creamy_tomato_italian_sausage_sauce/" target="_blank">Pasta Fettuccine</a></button>
+    <button id="penBtn"><a href="https://www.simplyrecipes.com/penne-alla-vodka-recipe-5324148/" target="_blank">Pasta Penne</a></button>
+    `
+  } else if (food === 'TexMex') {
+    botReply(`TexMex, huh? Muy picante! Please click on the dish you'd like to try.`)
+    inputWrapper.innerHTML = `
+    <button id="taqBtn"><a href="https://www.simplyrecipes.com/recipes/baked_chicken_taquitos/" target="_blank">Taquitos</a></button>
+    <button id="tacBtn"><a href="https://www.simplyrecipes.com/recipes/bbq_pulled_jackfruit_tacos/" target="_blank">Tacos</a></button>
+    <button id="chiBtn"><a href="https://www.simplyrecipes.com/recipes/moms_chili_beans/" target="_blank">Chili Beans with Rice</a></button>
+    `
+  } else if (food === 'Pizza') {
+    botReply(`Pizza, huh? Buon appetito! Please click on the dish you'd like to try.`)
+    inputWrapper.innerHTML = `
+    <button id="pepBtn"><a href="https://www.simplyrecipes.com/heart-shaped-pepperoni-pizza-recipe-6835896/" target="_blank">Heart-shaped Pepperoni Pizza</a></button>
+    <button id="musBtn"><a href="https://www.simplyrecipes.com/recipes/homemade_pizza/" target="_blank">Pizza with mushrooms</a></button>
+    <button id="hawBtn"><a href="https://www.simplyrecipes.com/recipes/hawaiian_pizza_with_cauliflower_crust/" target="_blank">Pizza Hawaii</a></button>
+    `
+// Had an else here, but then the next question kept skipping to this option
```

Comment:
I had problems with using else aswell, not sure why. But nice way to work around the problems we face.